ext/pgsql/tests/bugGH12763.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+Bug #GH12763 (pg_untrace(): Argument #1 ($connection) must be of type resource or null, PgSql\Connection given)
+--EXTENSIONS--
+pgsql
+--SKIPIF--
+<?php include("skipif.inc"); ?>
+--FILE--
+<?php
+include('config.inc');
+
+$conn = pg_connect($conn_str);
+
+pg_untrace($conn);
```

Comment:
file being named 'gh12763.phpt' should be good enough. I would like to see a bit more than this, involving pg_trace at least.